diff --git a/converter/adb_usb/Makefile b/converter/adb_usb/Makefile
index 14fc181b..b87f2e70 100644
--- a/converter/adb_usb/Makefile
+++ b/converter/adb_usb/Makefile
@@ -77,7 +77,7 @@ CONSOLE_ENABLE = yes	# Console for debug(+400)
 COMMAND_ENABLE = yes    # Commands for debug and configuration
 #SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
 #NKRO_ENABLE = yes	# USB Nkey Rollover
-ADB_MOUSE_ENABLE = yes
+# ADB_MOUSE_ENABLE = yes
 #UNIMAP_ENABLE = yes
 #ACTIONMAP_ENABLE = yes         # Use 16bit actionmap instead of 8bit keymap
 #KEYMAP_SECTION_ENABLE = yes	# fixed address keymap for keymap editor
