diff --git a/converter/adb_usb/config.h b/converter/adb_usb/config.h
index 3500188d..8d58ec80 100644
--- a/converter/adb_usb/config.h
+++ b/converter/adb_usb/config.h
@@ -41,10 +41,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 /* ADB port setting */
-#define ADB_PORT        PORTD
-#define ADB_PIN         PIND
-#define ADB_DDR         DDRD
-#define ADB_DATA_BIT    0
+#define ADB_PORT        PORTF
+#define ADB_PIN         PINF
+#define ADB_DDR         DDRF
+#define ADB_DATA_BIT    4
 //#define ADB_PSW_BIT     1       // optional
 
 /* key combination for command */
